src/makeCalculator.js

```diff
@@ -37,7 +37,33 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    operate(operation, number) {
+      this.result = operation(this.result, number);
+
+      return this;
+    },
+    add(result, number) {
+      return result + number;
+    },
```

Comment:
you can use result from your calculator

```suggestion
    add(number) {
      return calculator.result + number;
    },
```